build.gradle

```diff
@@ -12,8 +13,15 @@ repositories {
 
 def runeLiteVersion = '1.8.9'
 
+// Test locally via Gradle: "./gradlew run -DgimpDev=true"
```

Comment:
`./gradlew run -DgimpDev=true` is this supposed to have space => `./gradlew run -D gimpDev=true`?

Reply:
Nope. For some reason, Gradle System properties (`-Dsomething`) and project properties (`-Psomething`) don't have spaces between the CLI flag and the value. It's weird, but that's how it is.